build.gradle

```diff
@@ -1,28 +1,28 @@
-import java.util.stream.Collectors
-
 plugins {
     id 'java'
     id 'maven-publish'
-    id 'application'
-    id 'java-library'
     id 'signing'
 }
 
+if (System.getenv('RAPIDWRIGHT_PATH') == null) {
+  throw new GradleException('RAPIDWRIGHT_PATH environment variable not set')
```

Comment:
Although in the past we had recommended users set `RAPIDWRIGHT_PATH`, I've actually tried to remove the dependence of it.  We've had more trouble with users setting it and running into issues than it has solved.  Is there a way we can mimic what https://github.com/Xilinx/RapidWright/blob/567aeaf926381506e055aa9d37e154bed17a5304/src/com/xilinx/rapidwright/util/FileTools.java#L866 does when the variable is not set?